heapster-saw/src/Verifier/SAW/Heapster/SAWTranslation.hs

```diff
@@ -2529,12 +2543,19 @@ translateSimplImpl (ps0 :: Proxy ps0) mb_simpl m = case mbMatch mb_simpl of
            [nuP| DistPermsCons _ _ (ValPerm_LLVMArray mb_ap) |] -> return mb_ap
            _ -> error ("translateSimplImpl: SImpl_LLVMArrayFromBlock: "
                        ++ "unexpected form of output permission")
-       (w_term, _, elem_tp, ap_tp_trans) <- translateLLVMArrayPerm mb_ap
+       (w_tm, len_tm, elem_tp, ap_tp_trans) <- translateLLVMArrayPerm mb_ap
        withPermStackM id
          (\(pctx :>: ptrans_cell) ->
            let arr_term =
+                 -- FIXME: this generates a BVVec of length (bvNat n 1), whereas
+                 -- what we need is a BVVec of length [0,0,...,1]; the two are
+                 -- provably equal but not convertible in SAW core
+                 {-
                  applyOpenTermMulti (globalOpenTerm "Prelude.singletonBVVec")
-                 [w_term, elem_tp, transTerm1 ptrans_cell] in
+                 [w_tm, elem_tp, transTerm1 ptrans_cell]
+                 -}
+                 applyOpenTermMulti (globalOpenTerm "Prelude.repeatBVVec")
```

Comment:
Should we remove `singletonBVVec` from the Prelude now since it's no longer used?

Reply:
I dunno, I feel like we should leave it there as I guess a sort of documentation...?